pkgs/development/interpreters/python/hooks/stestr-check-hook.sh

```diff
@@ -0,0 +1,19 @@
+# Setup hook for stestr.
+# shellcheck shell=bash
+
+echo "Sourcing stestr-check-hook"
+
+stestrCheckPhase() {
+    echo "Executing stestrCheckPhase"
+    runHook preCheck
+
+    @pythonCheckInterpreter@ -m stestr run
```

Comment:
Can we also add an option to exclude specific tests? This is an example https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/python-modules/python-heatclient/default.nix#L70-L73